tests/unit/session-stores/adaptive-test.js

```diff
@@ -6,13 +6,18 @@ import Adaptive from 'ember-simple-auth/session-stores/adaptive';
 import itBehavesLikeAStore from './shared/store-behavior';
 import itBehavesLikeACookieStore from './shared/cookie-store-behavior';
 import FakeCookieService from '../../helpers/fake-cookie-service';
+import Configuration from 'ember-simple-auth/configuration';
 
 const { assign: emberAssign, merge, run } = Ember;
 const assign = emberAssign || merge;
 
 describe('AdaptiveStore', () => {
   let store;
 
+  beforeEach(function() {
+    Configuration.load({ baseURL: '/' });
```

Comment:
tests in `cookie-store-behaviour.js` expect that cookie path is set to '/' e.g.
https://github.com/simplabs/ember-simple-auth/blob/master/tests/unit/session-stores/shared/cookie-store-behavior.js#L110

I guess there are 2 ways to sort this out - either change it in all asserts from `path === '/'` to `path === Configuration.baseURL` or just set a default baseURL for all tests (which I did)